### DISCUSSION TITLE
Release notes issue vs PR based

### DISCUSSION THREAD

**Discussion by Sheikah45** [FAForever/downlords-faf-client#2383] [General]
Is there a reason why the release notes are issue based rather than merged PR based?

I am considering changing it to note the PRS that were merged and what their affects were rather than issue based as it seems more comprehensive but wanted to know if there was a reason for the initial choice.

**Comment by micheljung** [comment]
Issues are usually written by users and describe the Problem, sometimes including screenshots and discussions everyone can understand. Generally, issues are user-friendly.
PRs are created by developers and describe the solution. It's very technical and not user-friendly.

If someone creates or follows an issue he will then read in the release notes that the issue has been resolved. Also, everyone else can click on the resolved issue and read and understand the problem.
If we were to base the release notes on PRs, curious users would land on the PR which has all that strange stuff like code coverage, technical discussions etc. They're not helpful at all to them.

This is why I we use issues rather than PRs and I recommend sticking to it.
